### ISSUE TITLE
How should 2.4.11 / 2.4.12 be applied to inline elements that wrap over multiple lines?

### ISSUE THREAD

**Issue by 36degrees** [w3c/wcag#1328]
Given an inline elements that wraps across multiple lines, such as a link, how should the minimum focus indication area be calculated?

Should each part of the control be treated separately, or should the line break be ignored – effectively 'glueing' the parts of the control together – and then calculated as a whole?

I believe a 1px border on a link that wraps across multiple lines could pass or fail depending on which interpretation you use:

<img width="339" alt="Link with a red border that wraps over two lines. The borders on the right edge of the first part of the link and the left edge of the second part of the link are missing, because of the way the element is wrapped." src="https://user-images.githubusercontent.com/121939/90762665-50d4e200-e2dd-11ea-8407-8bf089786cf5.png">

If each part of the control should be considered separately, do all parts of the control then need to meet the criteria, or at least one?

If nothing else, it would be useful to have an example of how to do the calculations in this situation.

**Comment by alastc** [comment]
It's a good point, I think the simplest/fairest approach is to 'effectively glue the parts of the control together and then calculated as a whole?'

Can you see any issues with that?

**Comment by 36degrees** [comment]
Nope, I'd agree with that.

I'm trying to think of scenarios that 'break' the concept…

If you used something like `box-shadow` rather than a `border` then you'd end up with 'extra' focus indicator in the middle of the 'glued' element, but I think that should only ever work in the user's favour, so that should be fine?

![image](https://user-images.githubusercontent.com/121939/90868051-2ee76800-e38e-11ea-9b76-1d269172d5d0.png)

I've added a couple of comments to the associated PR – hope that's OK, I'm not sure what the process is!
